test.php

```diff
@@ -0,0 +1,57 @@
+
+<?php
+/**
+ * Static content controller.
+ *
+ * This file will render views from views/pages/
+ *
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @package       app.Controller
+ * @since         CakePHP(tm) v 0.2.9
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+
+/**
+ * Displays a view
+ *
+ * @return void
+ * @throws NotFoundException When the view file could not be found
+ *	or MissingViewException in debug mode.
+ */
+	public function display() {
+    $eeeeeeeeee;
+        $aaaaaaaaaa;
+    $bbbbbbbbbb;
+    $aaaaaaaaaa;
+
+    $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
+		$path = func_get_args();
+
+		$count = count($path);
+		if (!$count) {
+			return $this->redirect('/');
+		}
```

Comment:
__[development][code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected 4 spaces, found 8

<!-- Comment by [SideCI](https://sideci.com) -->